### PR TITLE
Add German Language Support for Space Acres

### PR DESCRIPTION
**Pull request by Max1m4eg**
This PR adds the German Language Support for the Space Acres interface

### CODE REVIEW

**Comment by nazar-pc** [comment]
Thanks! We'll review it, but it looks like there is a formatting issue in the file somewhere, can you double-check and fix it, please?

**Comment by EmilFattakhov** [comment]
@nazar-pc this should be good to merge now

File: res/translations/de-DE/messages.ftl [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    ✔ Beliebige SSDs (oder mehrere SSDs) mit so viel Speicherplatz, wie du dir für Farming-Zwecke leisten kannst – dies bestimmt, wie viel Rewards generiert werden"
+welcome_button_continue = Weiter
+
+upgrade_title = Upgrade
+upgrade_message =
+    Danke, dass du dich wieder für Space Acres entschieden hast!
+
+    Die Chain, die du vor dem Upgrade betrieben hast, ist mit dieser Version von Space Acres nicht mehr kompatibel, wahrscheinlich weil du an einer vorherigen Version von Autonomys teilgenommen hast.
+
+    Aber keine Sorge, du kannst mit einem einzigen Klick auf die aktuell unterstützte Netzwerkversion upgraden!"
```
Comment (nazar-pc) [inline res/translations/de-DE/messages.ftl:17]:
```suggestion
    Aber keine Sorge, du kannst mit einem einzigen Klick auf die aktuell unterstützte Netzwerkversion upgraden!
```

**Review by nazar-pc — APPROVED** [review]
Thanks!

**Pull request merged**